### ISSUE TITLE
Dimming lamps

### ISSUE THREAD

**Issue by TheBermud** [iobroker-community-adapters/ioBroker.hue#555]
I have previously taught the Philips Hue lamps via the Zigbee adapter. 
There was a brightness_move data point there. This made it very easy to dim the Hue lamps. Here is the description from the Zigbee adapter:Moving/stepping
Instead of setting a value (e.g. brightness) directly it is also possible to:

move: this will automatically move the value over time, to stop send value stop or 0.
step: this will increment/decrement the current value by the given one.
The direction of move and step can be either up or down, provide a negative value to move/step down, a positive value to move/step up. 

NOTE: brightness move/step will stop at the minimum brightness and won't turn on the light when it's off. In this case use brightness_move_onoff/brightness_step_onoff
  "brightness_move": -40, // Starts moving brightness down at 40 units per second
  "brightness_move": 0, // Stop moving brightness
  "brightness_step": 40 // Increases brightness by 40.

Is it possible to create a data point like this in the Hue adapter?

**Comment by foxriver76** [comment]
probably duplicate of https://github.com/iobroker-community-adapters/ioBroker.hue/issues/403

**Comment by TheBermud** [comment]
Yes, it definitely has something to do with it. But I don't think it's a duplicate. In my suggestion, the time in which the brightness changes is set, but no end value is set there or the brightness is changed up to the end value until you stop it. 

Oder hast du eine andere Idee wie ich die Lampen vernünftig dimmen kann mit den Datenpunkt die der Adapter zur Verfügung stellt?

**Comment by foxriver76** [comment]
Ja fürs dimmen ist normal transitiontime gedacht. Ich glaube es gibt keine DPs aktuell dafür siehe andere issues. Evtl geht es mit dem command Befehl, am besten mal Forum fragen ich nutze hier weniger intensiv

**Comment by TheBermud** [comment]
Vielen Dank für deine Antwort, ich frage dort nach.
